+dj/Relvar.m

```diff
@@ -215,35 +215,36 @@ function insert(self, tuples, command)
                 %   value:      <var> Processed, in-place value ready for insert.
                 %   placeholder:<string> Placeholder for argument substitution.
                 %   attr_idx:   <num> Attribute order index.
-                if header.attributes(attr_idx).isString
+                if isempty(value) || (header.attributes(attr_idx).isNumeric && isnan(value))
```

Comment:
What about an empty string? I will review this more carefully later tonight. But keep in mind that empty strings are distinct from `null`.

Reply:
Good point, I will review/test it once more too. How are nulls expected to return from a fetch for the common datatypes?

Reply:
Addressed